src/actions/conseillerActions.js

```diff
@@ -392,7 +392,9 @@ function preSelectionner(conseillerId) {
     .then(
       response => {
         dispatch(success(response.success));
-        dispatch(updateMiseEnRelation(response.miseEnRelation));
+        if (response?.miseEnRelation) {
```

Comment:
ok tu pourras même la brancher sur main directos puis merge back
A voir si dans cette même PR on corrige les autres sentry sur la pré selection (cannot read properties of undefined (reading '_id')